capsule-authentication.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/Platform/NVIDIA/Jetson/Jetson.dsc b/Platform/NVIDIA/Jetson/Jetson.dsc
+index 39e2c1d..3d6911d 100644
+--- a/Platform/NVIDIA/Jetson/Jetson.dsc
++++ b/Platform/NVIDIA/Jetson/Jetson.dsc
+@@ -33,3 +33,7 @@
+ 
+ [PcdsFixedAtBuild]
+   gNVIDIATokenSpaceGuid.PcdPlatformFamilyName|L"Jetson"
++!if $(CAPSULE_ENABLE)

```

Comment:
I'd rename this to something like CUSTOM_CAPSULE_CERT

Reply:
CAPSULE_ENABLE seems to be a convention used by boards in edk2-platforms, but the naming does overlap with just enabling capsule updates to begin with, not about capsule update authentication. Renamed to `CUSTOM_CAPSULE_CERT`